libs/dal/src/repositories/topic/topic-subscribers.entity.ts

```diff
@@ -18,4 +19,17 @@ export class TopicSubscribersEntity {
   externalSubscriberId: ExternalSubscriberId;
 }
 
+export type TopicSubscribersDBModel = Omit<
+  TopicSubscribersEntity,
+  '_environmentId' | '_organizationId' | '_subscriberId' | '_topicId'
+> & {
+  _environmentId: Types.ObjectId;
+
+  _organizationId: Types.ObjectId;
+
+  _subscriberId: Types.ObjectId;
+
+  _topicId: Types.ObjectId;
+};
```

Comment:
Would be so cool if we could implement a generic that could do this automatic type (pseudocode):
```
type CustomOmit<T, <A extends ReadonlyArray<string>, V = Types.ObjectId>> = Omit<T, A> & {
  [K in (A extends ReadonlyArray<infer A> ? A : never)]: V
}
``` 
So we could do something like this:
```
type TopicSubscriberDBModel = CustomOmit<TopicSubscribersEntity, <'_environmentId' | '_organizationId' | '_subscriberId' | '_topicId', Types.ObjectId>>;
```

Reply:
your wish is my order :D 
```
type ChangePropsValueType<T, K extends keyof T, V = Types.ObjectId> = Omit<T, K> & {
  [P in K]: V;
};

type TopicSubscribersDBModel = ChangePropsValueType<
  TopicSubscribersEntity,
  '_environmentId' | '_organizationId' | '_subscriberId' | '_topicId'
>;
```